sys-firmware/broadcom-bt-firmware/broadcom-bt-firmware-12.0.1.1105_p4-r1.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit linux-info multiprocessing
+
+DESCRIPTION="Broadcom Bluetooth firmware"
+HOMEPAGE="https://github.com/winterheart/broadcom-bt-firmware"
+SRC_URI="https://github.com/winterheart/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="broadcom_bcm20702 MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="compress-xz compress-zstd"
```

Comment:
I get the feeling that you deliberated named the USE flags `compress-*`. Could you elaborate why you went with the `compress-` prefix, and not just re-used, for example, the existing `zstd` use flag?

Reply:
These USE flags same as on recent `sys-kernel/linux-firmware`.

Reply:
Thanks, maybe a good idea to add a comment about that.
```suggestion
# Re-use compress-* USE flags from sys-kernel/linux-firmware.
IUSE="compress-xz compress-zstd"
```